### PR TITLE
Full migration to Lagom 1.5.0-RC2

### PR DESCRIPTION
**Pull request by octonato**
* Lagom takes care of cluster bootstrap
* health-check is included by default
* updated dependencies: akka-management 1.0.0, akka service-locator 0.1.0

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -66,11 +64,9 @@ lazy val inventory = (project in file("inventory"))
 val lombok = "org.projectlombok" % "lombok" % "1.16.18"
 val postgresDriver = "org.postgresql" % "postgresql" % "42.2.5"
 val hamcrestLibrary = "org.hamcrest" % "hamcrest-library" % "2.1" % Test
-val akkaDiscoveryServiceLocator = "com.lightbend.lagom" %% "lagom-javadsl-akka-discovery-service-locator" % "0.0.12"
+val akkaDiscoveryServiceLocator = "com.lightbend.lagom" %% "lagom-javadsl-akka-discovery-service-locator" % "0.1.0"
```
Comment (ignasi35) [inline build.sbt:67]: nice!

File: shopping-cart/src/main/java/com/example/shoppingcart/impl/ShoppingCartModule.java [hunk 2]
```diff
@@ -12,6 +12,5 @@
     @Override
     protected void configure() {
         bindService(ShoppingCartService.class, ShoppingCartServiceImpl.class);
-        bind(ClusterBootstrapStart.class).asEagerSingleton();
```
Comment (ignasi35) [inline shopping-cart/src/main/java/com/example/shoppingcart/impl/ShoppingCartModule.java:15]: Wasn't the `-java-` sample app starting anything else? Weird. I think the `-scala-` also started `AkkaManagement` but I guess it happens transitively.

**Review by ignasi35 — APPROVED** [review]

File: shopping-cart/src/main/java/com/example/shoppingcart/impl/ShoppingCartModule.java (hunk 2 above)
Comment (octonato) [reply shopping-cart/src/main/java/com/example/shoppingcart/impl/ShoppingCartModule.java:15]: I don't know if I get your point. 

`ClusterBootstrapStart` was a custom component that was eargely starting akka-management and cluster bootstrap. This is not needed in LAgom 1.5.0, so I remove it.

Comment (ignasi35) [reply shopping-cart/src/main/java/com/example/shoppingcart/impl/ShoppingCartModule.java:15]: I confused `ClusterBootstrapStart` with [`ClusterBootstrat().start()`+`AkkaManagement.start()`](https://github.com/lagom/shopping-cart-scala/pull/17/files#diff-1cddafe3b105a076fbc4b457f7a0b6a7L20) so I didn't understand why we _only_ needed to remove one line of code.

:-)

**Pull request merged**